features/other_b/skin.feature

```diff
@@ -491,16 +207,43 @@ Feature: creating and editing skins
   When I press "Use"
   Then the page should have a skin with the media query "only screen and (max-width: 42em), only screen and (max-width: 62em)"
 
-  Scenario: A user should be able to choose a temporary skin
-  Given basic skins
-    And the approved public skin "public skin" with css "#title { text-decoration: blink;}"
-    And the skin "public skin" is cached
-    And the skin "public skin" is in the chooser
+  Scenario: A user should be able to delete a skin
+  Given I am logged in as "skinner"
+    And I create the skin "Ugly Skin"
+  When I go to "Ugly Skin" skin page
+    And I follow "Delete"
+  Then I should see "The skin was deleted."
+    And I should be on skinner's skins page
```

Comment:
Is it worth adding a check that the skin is not a visible on the skinners' skin's page ?